src/components/sound-card/sound-card.jsx

```diff
@@ -0,0 +1,31 @@
+import { useSounds } from "../../hooks/use-sounds";
+import purpleGoblin from "../../images/purple-goblin.png";
+
+export function SoundCard({ soundInfo }) {
+  const card = {
+    image: soundInfo?.photo,
+    title: soundInfo?.title,
+    audio: soundInfo?.audio,
+  };
+
+  const { playSound } = useSounds();
+
+  function handleSoundClick() {
+    console.log(card.audio);
+    playSound("https://" + card.audio);
+  }
+
+  return (
+    <img
+      className={
+        "flex-none overflow-hidden hover:cursor-pointer object-cover h-[130px] w-[130px] self-baseline xs:self-center"
+      }
+      src={!card.image ? purpleGoblin : card.image?.url}
```

Comment:
nit: Instead of checking if we don't have the object, we can flip the logic here for better readability: 'if we have the image use the image, else default to purple goblin'.